controllers/exclude_instances.go

```diff
@@ -55,7 +55,7 @@ func (e ExcludeInstances) Reconcile(r *FoundationDBClusterReconciler, context ct
 
 		for _, state := range cluster.Status.PendingRemovals {
 			if state.Address != "" {
-				address := RemoveAddressFlags(cluster.GetFullAddress(state.Address))
+				address := state.Address
```

Comment:
We should do a test to make sure the transition between exclusion methods is safe:

1. Start a replacement on the old code
2. Let the replacement get far enough to start the exclusion
3. Update the operator to use the new code
4. Let the replacement finish on the new code.
5. Confirm that exclusion checks work fine, and that the inclusion command cleans up everything in the exclusion list.

Reply:
In my tests everything worked as expected. I ran the tests manually since we currently don't have any mechanism to run these tests automatically.